csi/moac/src/csi.ts

```diff
@@ -550,42 +553,29 @@ export class CsiServer {
         // standing up a volume, using the volume context.
         volumeContext: args.parameters
       }
-    });
+    };
   }
 
-  async deleteVolume (args: any, cb: CsiDoneCb) {
+  async deleteVolume (args: any) {
     assert(this.volumes);
 
     log.debug(`Request to destroy volume "${args.volumeId}"`);
 
-    // If this is a duplicate request then assure it is executed just once.
```

Comment:
Do we not need to check for duplicate requests for deletions now?

Reply:
Checking for duplicate requests makes never harm. The assumption we make is that if there is the same method with the same parameters already in progress, we wait for its completion and return the same reply as for the operation which is in progress. Because all methods should be idempotent it is not strictly necessary to detect duplicates but it saves some time and reduces log cluttering. It is also a good performance optimization because k8s can be quite aggressive with sending retries. So it is not strictly necessary for any of the methods but we do it for all.